tox.ini

```diff
@@ -71,6 +71,8 @@ deps = {[testenv:lint]deps}
 commands = pytest {toxinidir}/tests/unit \
   {posargs:-v \
            --cov=src \
+           --cov-fail-under 100 \
+           --cov-report=xml \
            --cov-report=term-missing \
            --cov-report=html:report/html}
```

Comment:
We had a few discussions in the past weeks about the standard way of configuring pytest. As an outcome, the desirable approach is changing these lines to the following:
```
            --cov \
            --cov-report=term \
            --cov-report=html \
            --cov-report=xml}
```
And then in pyproject.py, add the configuration for the coverage scope and reports ( https://git.launchpad.net/bootstack-charms-spec/tree/source/snap-pytest/pyproject.toml#n48). 

These should be updated in the template folder for the charms.

Reply:
I think you meant `pyrpoject.toml`. We agreed that this 
```ini
{posargs:-v \
               --cov \
               --cov-report=term \
               --cov-report=html \
               --cov-report=xml}
```
It's good approach, however `posargs` could not support multi-lines default value so we need to use:
```ini
{posargs:-v  --cov  --cov-report=term  --cov-report=html  --cov-report=xml}
```

Reply:
thanks. I'll reuse pyproject.toml referenced by @agileshaw with slight changes. In this case, I think we can change `source` in `[tool.coverage.run]` to just `["src/"]` and we don't have to ommit anything.